estate_account/models/estate_property.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, Command
+
+
+class EstateProperty(models.Model):
+    # ---------------------------------------- Private Attributes ---------------------------------
+
+    _inherit = "estate.property"
+
+    # ---------------------------------------- Action Methods -------------------------------------
+
+    def action_sold(self):
+        res = super().action_sold()
+        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
+        for record in self:
+            values = {
+                "partner_id": record.buyer_id.id,
+                "move_type": "out_invoice",
+                "journal_id": journal.id,
+                "invoice_line_ids": [
+                    Command.create(
+                        {
+                            "name": record.name,
+                            "quantity": 1.0,
+                            "price_unit": record.selling_price * 6.0 / 100.0,
+                        }
+                    ),
+                    Command.create(
+                        {
+                            "name": "Administrative fees",
+                            "quantity": 1.0,
+                            "price_unit": 100.0,
+                        }
+                    ),
```

Comment:
```suggestion
                    self.create_line(record.name, 1.0, record.selling_price * 6.0 / 100.0),
                    self.create_line("Administrative fees", 1.0, 100.0),
```

We can move this logic to a function called `create_line()`